lib/src/auto_injector_base.dart

```diff
@@ -185,6 +185,7 @@ abstract class AutoInjector extends Injector {
 
 class _AutoInjector extends AutoInjector {
   final _binds = <Bind>[];
+  final _injectorsList = <_AutoInjector>[];
```

Comment:
It's not useful anymore because now we are using the **LayersGraph** to find binds and injectors using Graph with [BFS (Breadth First Search)](https://www.geeksforgeeks.org/breadth-first-search-or-bfs-for-a-graph/).

This list only deserves to help **LayersGraph** fill the `adjacencyList`  when `LayersGraph.initialize()` is called.